kylin-spark-project/kylin-spark-engine/src/main/java/org/apache/kylin/engine/spark/job/NSparkExecutable.java

```diff
@@ -105,7 +105,7 @@ protected ExecuteResult doWork(ExecutableContext context) throws ExecuteExceptio
         //context.setLogPath(getSparkDriverLogHdfsPath(context.getConfig()));
         CubeManager cubeMgr = CubeManager.getInstance(KylinConfig.getInstanceFromEnv());
         CubeInstance cube = cubeMgr.getCube(this.getCubeName());
-        KylinConfig config = cube.getConfig();
+        KylinConfig config = KylinConfigExt.createInstance(cube.getConfig(), cube.getDescriptor().getOverrideKylinProps());
```

Comment:
Don't need to create a KylinConfigExt here, the returned value of 'cube.getConfig()' already is a KylinConfigExt object.

Reply:
thansks you for your advice ~